src/wasm/wasm-validator.cpp

```diff
@@ -862,6 +862,7 @@ void FunctionValidator::validateAlignment(size_t align, Type type, Index bytes,
       shouldBeTrue(align <= 8, curr, "alignment must not exceed natural");
       break;
     }
+    case v128: assert(false && "v128 not implemented yet");
```

Comment:
can we not just say `align <= 16` here? (how does simd alignment work in wasm?)

Reply:
Yes, essentially. The natural alignment of vectors is 16 bytes, and any smaller power of two alignment is also allowed. This will be implemented in a future PR.